src/controllers/users.controllers.js

```diff
@@ -0,0 +1,15 @@
+import User from "../services/user.services";
+
+
+export default class Users {
+  static async signup(req, res) {
+    try {
+      let { data } = await User.register(req.body)
+      return res.status(200).json({ message: "user created successful", data});
+    }
+    catch(e) {
+      console.log(e)
```

Comment:
remove console.log, it should be for debugging purposes only, otherwise it should not be pushed